### DISCUSSION TITLE
GitHub sponsor

### DISCUSSION THREAD

**Discussion by marcinkuzminski** [bashly-framework/bashly#348] [General]
Hi,

Thank you for this great project!
Would you consider opening this for sponsorship in github ? My company would like to give some gratitude for this project, and how it helped us :)

Keep up the great work!

**Comment by DannyBen** [comment]
Marcin - you made my day already.

It is very nice to see that the project I developed out of my personal need is used by others, and even in a business context.
I will setup a GitHub sponsors account and post here when it is approved.

I appreciate the gesture, and regardless - if you have questions about bashly, or if some things are not working as expected - please let me know.

**Comment by marcinkuzminski** [comment]
Thanks, we will for sure, so far it is exactly what we needed to build our new CLI for installing our product called RhodeCode.

I'll check once the sponsorship is ready.

Cheers,

**Reply by DannyBen** [reply]
Well - that was faster than anticipated :)
Sponsorship is enabled on the Bashly repository - thanks for suggesting it, and for your willingness - it is appreciated.

**Reply by DannyBen** [reply]
@marcinkuzminski - I don't know what to say... a huge thank you for a jaw-dropping moment.
I will add the RhodeCode logo and link to the site tomorrow - please email me if you wish it to be something else, and for anything else I can help with.

**Reply by marcinkuzminski** [reply]
@DannyBen thanks! I'm happy that we could give back to the open-source community, and thank you for your work !
If you need here are the logos: https://rhodecode.com/press

Cheers !
